### PR TITLE
[Naitei][Ecommerce]    [User] Like product

### PR DESCRIPTION
**Pull request by SuKay712**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/78391

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)

https://github.com/user-attachments/assets/

https://github.com/user-attachments/assets/86c98ec0-92a9-43f0-a895-0660dc187596

ba489071-e90e-4f93-95d9-67fbf5d43358

### CODE REVIEW

**Comment by SuKay712** [comment]
ready

File: app/controllers/wishlists_controller.rb [hunk 1]
```diff
@@ -1,7 +1,34 @@
 class WishlistsController < ApplicationController
+  before_action :load_product, only: %i(create destroy)
```
Comment (vanvtt-0952) [inline app/controllers/wishlists_controller.rb]: cho hỏi có check user đã login chưa sư huynh

File: app/controllers/wishlists_controller.rb [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
   def index
     @wishlists = current_user.wishlists
   end
 
-  def create; end
+  def create
+    current_user.like(@product)
+
+    respond_to do |format|
+      format.html{redirect_to @product}
```
Comment (vanvtt-0952) [inline app/controllers/wishlists_controller.rb]: hay đó, cơ mà thêm cái flash cho thân thiện đi cụ, chứ im ỉm ri sao bjk dc

File: app/controllers/wishlists_controller.rb (hunk 1 above)
Comment (SuKay712) [reply app/controllers/wishlists_controller.rb]: dậa

File: app/controllers/wishlists_controller.rb (hunk 2 above)
Comment (SuKay712) [reply app/controllers/wishlists_controller.rb]: dạ oke rùi ạ

**Comment by SuKay712** [comment]
ready

**Review by ghost — COMMENTED** [review]
confflict rồi em

**Comment by vanvtt-0952** [comment]
commented
chưa review nhưng conflict code thì ... xứng đáng thu tiền

**Comment by SuKay712** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**